.clang-tidy

```diff
@@ -35,7 +35,6 @@ Checks: >
   -modernize-use-auto,
   -modernize-use-nodiscard,
   -modernize-use-trailing-return-type,
-  -modernize-use-using,
```

Comment:
Why do you remove it?

Reply:
Now that this PR has removed all the typedefs, we should enable this check to make it harder for anyone to re-introduce them.

Reply:
Ah, now it's clear - it was disabled earlier. But what about third-party code "shipped' with qBittorrent?

Reply:
To confirm, your concern is that this warning will fire for the 3rd-party code, but the warning would be superfluous since we don't care about fixing 3rd-party code?

In that case we could a .clang-tidy to the 3rd-party directory that disables all checks there (`Checks: -*`)

Reply:
> In that case we could a .clang-tidy to the 3rd-party directory that disables all checks there (`Checks: -*`)

:+1: